src/components/Blog/Post/Markdown/styles.module.css

```diff
@@ -631,3 +631,50 @@
     }
   }
 }
+
+.admonition {
```

Comment:
Here's how the design looks so far: 

<img width="773" alt="image" src="https://user-images.githubusercontent.com/43496356/159074436-2019565a-c7e1-4b74-9d51-3d9d80571b4b.png">

Is it good enough for a first iteration? cc @yalozhkin

Reply:
Looks good, but I find it to be too flat. details block and code-blocks have rounded corners. Would it be possible to make it look like details block?

Reply:
I like the rounded corners too!

Reply:
> Looks good, but I find it to be too flat. details block and code-blocks have rounded corners. Would it be possible to make it look like details block?
> I like the rounded corners! 

Thanks for the feedback! I updated the design to have rounded corners: 

<img width="778" alt="image" src="https://user-images.githubusercontent.com/43496356/159259133-190fc8a2-2b75-4738-bf6d-4e99ee7c0824.png">


I took off the border for now, but if we wanted to keep it, here is what is would look like:

<img width="766" alt="image" src="https://user-images.githubusercontent.com/43496356/159259097-20868fa4-7fe8-49d6-ae01-96d5f12d8eb6.png">

Reply:
I like the border

Reply:
➕ to the border!

Reply:
I like the border too

Reply:
Late +1 for border+rounded, looks nice!